test/Datadog.Trace.TestHelpers/MockTracerAgent.cs

```diff
@@ -146,42 +146,6 @@ protected virtual void OnRequestReceived(HttpListenerContext context)
             RequestReceived?.Invoke(this, new EventArgs<HttpListenerContext>(context));
         }
 
-        private static List<Span> ToSpans(dynamic data)
```

Comment:
Should we merge the PR that this come from first so this (unrelated) diff is not here?